### PR TITLE
Surf_reader : Enhancement

### PR DESCRIPTION
**Pull request by maxGimeno**
## Summary of Changes
Enhancement for our surf_reader
## Release Management

* Affected package(s):Polyhedron_demo

### CODE REVIEW

**Comment by janetournois** [comment]
It works for me, thank you!

**Comment by janetournois** [comment]
Hi  @maxGimeno 

there is another problem with our surf reader and the input file I gave you.
The file is read and the demo displays the polyhedral surface, but inside out, and the console displays :

```
732 vertices
1 patch(es)
The patch #0:
  -inner region : material's id = 0 material's name = Exterior
  -outer region: material's id = 1 material's name = Material1
```

Can you please make sure "exterior" corresponds to "outer region"?

**Comment by maxGimeno** [comment]
Successfully tested in https://cgal.geometryfactory.com/CGAL/testsuite/results-5.1-Ic-133.shtml

File: Polyhedron/demo/Polyhedron/include/CGAL/IO/read_surf_trianglemesh.h [hunk 1]
```diff
@@ -72,6 +72,208 @@ bool line_starts_with(const std::string& line, const char* cstr)
...
+                         const std::string& input_line,
+                         std::size_t& nb_vertices,
+                         std::vector<Point_3>& points)
+{
+  std::istringstream iss;
+  iss.str(input_line);
+  std::string dump;
+  iss >> dump >> nb_vertices;
+  points.reserve(nb_vertices);
+  std::cout<<nb_vertices<<" vertices"<<std::endl;
```
Comment (sloriot) [inline Polyhedron/demo/Polyhedron/include/CGAL/IO/read_surf_trianglemesh.h:128]: is it on purpose ?

File: Polyhedron/demo/Polyhedron/include/CGAL/IO/read_surf_trianglemesh.h [hunk 2]
```diff
@@ -72,6 +72,208 @@ bool line_starts_with(const std::string& line, const char* cstr)
...
+                      std::vector<Mesh>& output,
+                      std::vector<Point_3> points,
+                      std::vector<std::array<std::size_t, 3> >& polygons,
+                      const int& i)
+{
+  namespace PMP = CGAL::Polygon_mesh_processing;
+  typedef std::array<std::size_t, 3> Triangle_ind;
+  if (!PMP::is_polygon_soup_a_polygon_mesh(polygons))
+  {
+    std::cout << "Orientation of patch #" << (i + 1) << "...";
```
Comment (sloriot) [inline Polyhedron/demo/Polyhedron/include/CGAL/IO/read_surf_trianglemesh.h:246]: same question here

File: Polyhedron/demo/Polyhedron/include/CGAL/IO/read_surf_trianglemesh.h (hunk 1 above)
Comment (maxGimeno) [reply Polyhedron/demo/Polyhedron/include/CGAL/IO/read_surf_trianglemesh.h:128]: Yes, it was there before.

File: Polyhedron/demo/Polyhedron/include/CGAL/IO/read_surf_trianglemesh.h (hunk 2 above)
Comment (maxGimeno) [reply Polyhedron/demo/Polyhedron/include/CGAL/IO/read_surf_trianglemesh.h:246]: Yes, it is a useful information.

**Pull request merged**